### DISCUSSION TITLE
[Script request] OnlyOffice

### DISCUSSION THREAD

**Discussion by ma-gh** [community-scripts/ProxmoxVE#408] [Request script]
### Application Name

OnlyOffice

### Website

https://www.onlyoffice.com/download-docs.aspx?from=officefornextcloud#docs-community

### Description

OnlyOffice for integration with existing script NextCloud

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by fbloemhof** [comment]
I was curious for the same thing. Found out that I was able to install NextCloud Office and Collabora Server as apps inside Nextcloud. Not sure if this suits your needs but it might be something.

**Comment by floppymaster** [comment]
W8ing to came true...

Thanx for the effort put in this!

**Comment by lightstal** [comment]
Hope this comes soon!

**Comment by varunkotian** [comment]
Waiting for this!

**Comment by MickLesk** [comment]
The onlyoffice script is ready for testing:
```bash -c "$(curl -fsSL https://github.com/community-scripts/ProxmoxVED/raw/main/ct/onlyoffice.sh)"```
Note: This is not in the official repo yet—it's just a dev version! After merging into ProxmoxVE, it will need to be recreated.


=> https://github.com/community-scripts/ProxmoxVED/issues/579
